docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,441 @@
+---
+id: server-binary-tutorial
+sidebar_position: 1
+keywords: [sqlite, nginx, binary, systemd, https, certbot, hosting, deploy, config, minimal, baseline]
+tags: [Server, Binary]
+last_update:
+  date: 2024-07-08
+title: How to Deploy a Temporal Service using an SQLite Backend without Docker or Kubernetes
+description: Deploy a Temporal Service from scratch using our Server Binaries without requiring any additional dependencies.
+image: /img/temporal-logo-twitter-card.png
+---
+
+### Introduction
+
+There are many ways of deploying a Temporal Service. For a large-scale deployment, you can use [Docker](https://github.com/temporalio/docker-compose) or [Kubernetes](https://github.com/temporalio/helm-charts) to configure multiple pods with the ability to scale horizontally. For local development, you can use the `server` subcommand of the [Temporal CLI client](https://docs.temporal.io/cli/server) to run a single-user server.
+
+If you need a deployment that fits in between these options -- for example, if you need to scale for multiple users, with fine-grained control over your deployment parameters, but without the overhead of Kubernetes -- you can deploy a Temporal Service using the official server binaries.
+
+In this tutorial, you'll learn how to configure and deploy the two binaries needed for a complete Temporal Service (the core server and the UI server). You'll create `systemd` unit files to gracefully run and restart the Temporal Service automatically upon server startup, and you'll deploy an Nginx reverse proxy to handle web traffic ingress. This will give you everything you need to run a production Temporal Service, and evaluate how to scale further or [migrate to Temporal Cloud](https://temporal.io/cloud). Let's get started.
+
+## Prerequisites
+
+- A Linux server with SSH access and the Nginx web server installed. This can be a new Ubuntu server instance with no additional configuration performed. However, you will need a version of Nginx built with HTTP/2 support (at least version 1.25.1), which may not be available by default in some environments. On Ubuntu, you can use [this PPA](https://launchpad.net/~ondrej/+archive/ubuntu/nginx) to install a compatible Nginx.
+- To enable HTTPS in the browser, you will need SSL certificates and your own domain name pointing to the server.
```

Comment:
```suggestion
- To enable HTTPS in the browser, you will need SSL certificates and your own domain name pointing to the server.

Once you have your domain pointing the the server, one option to get an SSL certificate is to use Certbot. This tutorial will assume this was the option you chose. Use the commands below, replacing `YOUR_DOMAIN` with the domain pointing to your server to generate the certificate:

```bash
systemctl stop nginx # ensure that nginx is stopped and port 80 is available
sudo snap install --classic certbot
sudo ln -s /snap/bin/certbot /usr/bin/certbot
sudo certbot certonly --standalone -d YOUR_DOMAIN
```

```